stages/14-director-gui-check/gui-dmaas-cstor-non-restic

```diff
@@ -15,7 +15,7 @@ cp  $path/.awscluster2/config ~/.kube/config
 ####################################
 ##  Sequencing and Running test   ##
 ####################################
-bash utils/pooling jobname:tcid-dir-dmaas-schd-cstor-aws-bucket
+bash utils/pooling jobname:tcid-dir-dmaas-schd-local-dev-aws-bucket
```

Comment:
These line of code is too difficult to understand

Reply:
above code is for sequencing of jobs.
`tcid-dir-dmaas-schd-local-dev-aws-bucket` job is waiting for previous job `tcid-dir-dmaas-schd-cstor-aws-bucket` to complete.